tests/scripts/analyze_outcomes.py

```diff
@@ -624,21 +638,7 @@ def do_analyze_driver_vs_reference(results: Results, outcome_file, args):
                 ],
                 'test_suite_asn1write': [
                     # Following tests depends on BIGNUM_C
-                    'ASN.1 Write mpi 0 (1 limb)',
-                    'ASN.1 Write mpi 0 (null)',
-                    'ASN.1 Write mpi 0x100',
-                    'ASN.1 Write mpi 0x7f',
-                    'ASN.1 Write mpi 0x7f with leading 0 limb',
-                    'ASN.1 Write mpi 0x80',
-                    'ASN.1 Write mpi 0x80 with leading 0 limb',
-                    'ASN.1 Write mpi 0xff',
-                    'ASN.1 Write mpi 1',
-                    'ASN.1 Write mpi, 127*8 bits',
-                    'ASN.1 Write mpi, 127*8+1 bits',
-                    'ASN.1 Write mpi, 127*8-1 bits',
-                    'ASN.1 Write mpi, 255*8 bits',
-                    'ASN.1 Write mpi, 255*8-1 bits',
-                    'ASN.1 Write mpi, 256*8-1 bits',
+                    re.compile('ASN.1 Write mpi.*'),
```

Comment:
Non-blocker: it's good to get into the habit of writing `re.compile(r'…')`, because regexes very often contain backslashes. It only matters if the regex actually contains a backslash though.

Reply:
Fun story: I got told off by pylint because I was using backslashes in a weird way in some regexes, which I fixed by using `r'...'` for those. You're right, I should have gone all the way and used `r` everywhere.